app/views/projects/automated_workflow_executions/_pipeline_selection_modal.html.erb

```diff
@@ -0,0 +1,60 @@
+<%= viral_dialog(open: open) do |dialog| %>
+  <%= dialog.with_header(title: t(:".title")) %>
+  <%= dialog.with_section do %>
+    <ul>
+      <% available_automated_workflows.each do |key, workflow| %>
+        <li>
+          <%= link_to new_namespace_project_automated_workflow_execution_path(@project.parent, @project, workflow_name: workflow.name, workflow_version: workflow.version), class: "group w-full", data: { turbo_stream: true } do %>
+            <span
+              class="
+                ws-default
+                flex
+                items-center
+                p-4
+                space-x-4
+                hover:bg-primary-600
+                btn
```

Comment:
**nitpick:** Can you replace `btn` with the actual flowbite classes.  Where did you copy that from?

Reply:
Removed in f9dea938a9deda9d83d0218f1b7a935f247bc25a

Reply:
Thanks